### DISCUSSION TITLE
シェル芸ボットでどんな危険シェル芸を試すべきか

### DISCUSSION THREAD

**Discussion by butackle** [shellgei/shellgei-bot.js#7] [General]
危険シェル芸の提案をお願いします。
こんな感じのことを書いてもらえれば多分大丈夫です。

- 実行するシェル芸
- どんな効果・ダメージがあるか

**Comment by ryuichiueda** [comment]
: () { : | :  & } ; : 

プロセスが倍々に増えてホストごと固まります。

**Comment by greymd** [comment]
cat /dev/zero > a
ダメージあるかわかりませんが、とりあえずデカいファイルを作成してホストを苦しめてみる

**Comment by greymd** [comment]
自身をシンボリックリンクにして別のファイルを指す。なにかおきそう。

```
rm $0
ln -s /etc/ホスト上の大事なファイルとか？ $0
```

**Comment by yabeenico** [comment]
カーネルパニック

```
echo 1 >/proc/sys/kernel/sysrq
echo c >/proc/sysrq-trigger
```
